internal/service/elasticache/replication_group.go

```diff
@@ -677,6 +698,9 @@ func resourceReplicationGroupRead(d *schema.ResourceData, meta interface{}) erro
 	d.Set("arn", rgp.ARN)
 	d.Set("data_tiering_enabled", aws.StringValue(rgp.DataTiering) == elasticache.DataTieringStatusEnabled)
 
```

Comment:
you can remove the whiteline above and below the newly added attributes.

Reply:
Completed